azure/route-table/README.md

```diff
@@ -0,0 +1,46 @@
+# Terraform (Module) - Azure - Route Table
+
+#### Table of Contents
+
+- [Terraform (Module) - Azure - NAME](#terraform-module---azure---name)
+      - [Table of Contents](#table-of-contents)
+  - [Usage](#usage)
+  - [Requirements](#requirements)
+  - [Inputs](#inputs)
+  - [Outputs](#outputs)
+  - [Resources](#resources)
+  - [Modules](#modules)
+
+## Usage
+
+This module creates a route table, routes and associates any subnet ids provided.
+
+There is the option to create a "default route" which routes all IPs (0.0.0.0/32)to a specified default_route_ip.  If no IP is specified, no route is created.
```

Comment:
All IPs are `0.0.0.0/0` not `0.0.0.0/32` - `/32` denotes a single address not the whole range

https://en.wikipedia.org/wiki/0.0.0.0#Routing

Also missing a space between `)` and `t`